src/main/java/macrobase/analysis/result/OutlierClassificationResult.java

```diff
@@ -0,0 +1,4 @@
+package macrobase.analysis.result;
+
+public abstract class OutlierClassificationResult {
```

Comment:
I think we'll probably have just one class implementing this, but I like having an abstract class for each of the components, then we have a clear API that is visible. Also for this stage I didn't seem to have an alternative, so if you feel strongly we can just convert this into a normal class in the future